packages/webapp/src/routes/land/upload-ownership-proof.js

```diff
@@ -13,7 +13,7 @@ const LAND_OWNERSHIP_ID = '#landOwnership'
 async function processSuccessfulUpload (result, request, h) {
   const tempFile = request.yar.get(constants.redisKeys.TEMP_LAND_OWNERSHIP_PROOF)
   if (tempFile && !tempFile.confirmed) {
-    await deleteBlobFromContainers(tempFile.fileLocation)
+    await deleteBlobFromContainersWithCheck(tempFile.fileLocation)
```

Comment:
Only one blob path is being passed here. My reading of the code is that it should be:
```js
await deleteBlobFromContainersWithCheck(result.config.blobConfig.blobname, tempFile.fileLocation)
```

But I could be wrong here!